proto/canal/CanalProtocol.pb.go

```diff
@@ -5,16 +5,19 @@ package com_alibaba_otter_canal_protocol
 
 import (
```

Comment:
I suggest not to modify these generated pb files until https://github.com/pingcap/ticdc/issues/1107 is fixed.